calculator.js

```diff
@@ -38,8 +36,19 @@ async function getOperation() {
 }
 
 async function getOperand() {
-  let Operand = parseFloat(await rl.question("Input Operand: "));
-  return Operand;
+  let isSelectingOperand = true;
+  
+  while (isSelectingOperand) {
+    let Operand = parseFloat(await rl.question("Input Operand: "));
+    if (!(isNaN(Operand))) {
+    isSelectingOperand = false;
```

Comment:
Indentation needs to be fixed. Anything inside {} should be indented over by one

```
if () {
  // always indent here like this
// not like this
}
```